libs/dynamodb-lib.js

```diff
@@ -0,0 +1,10 @@
+import AWS from "aws-sdk";
+
+/**
+ * Bootstrap and provide method to call DynamoDB.
+ */
+export function call(action, params) {
+  const dynamoDb = new AWS.DynamoDB.DocumentClient();
+
+  return dynamoDb[action](params).promise();
```

Comment:
Hypothetically, speaking, how would you recommend extending this code if the dynamoDB client began name-spacing features/versions under object properties?

For example, if the `someAction` method existed that also needed to be accessed here: `dynamoDb.v2.someAction(params).promise`

Reply:
If the caller needed to explicitly select a versioned API I'd look to add in a version parameter or some agreed upon best practice to allow the caller to select the version they prefer.  In a more ideal situation, where the caller may not need to know or care, it could be encapsulated inside the method.

I think what is important is looking at future migration, do we want our callers to have to migrate their code to a new version (as they are specifying which version they are calling) or do we want to control what version is being called so as to respect a basic interface contract the clients are calling.  This may be a case by case situation.